README.md

```diff
@@ -1,6 +1,6 @@
 # BroadwayKafka
 
-[![Build Status](https://github.com/dashbitco/broadway_kafka/workflows/CI/badge.svg)](https://github.com/dahsbitco/broadway_kafka/actions)
```

Comment:
The `dahsbitco` mispelling was what tipped me off about this, but I realized the SVG URL also didn't match what I typically see.